openproblems/tasks/_batch_integration/batch_integration_graph/metrics/iso_label_f1.py

```diff
@@ -37,7 +37,7 @@ def isolated_labels_f1(adata):
         adata,
         label_key="labels",
         batch_key="batch",
-        embed="X_pca",
+        embed=None,
```

Comment:
Can you explain this change? Was it a bug? Doesn't seem explicitly related to the PR

Reply:
Yes, this was a bug. This recalculates the neighborhood graph using the provided embedding, which I don't think we want to do for the graph task